tasks/omnibus.py

```diff
@@ -379,3 +379,26 @@ def manifest(
         omnibus_s3_cache=False,
         log_level=log_level,
     )
+
+@task
+def rpath_edit(ctx, install_path, target_rpath_dd_folder, macos=False):
+    for file in glob.iglob(f"{install_path}/**/*", recursive=True):
+        ext = os.path.splitext(file)[1]
+        if (ext != "" and "so" not in ext and "dylib" not in ext) or "json" in ext or not os.path.isfile(file):
+            continue
+        if macos:
+            toedit_fd = ctx.run(f'otool -l {file} | grep -A 2 "RPATH"', warn=True, hide=True)
+        else:
+            toedit_fd = ctx.run(f'objdump -x {file} | grep "RPATH"', warn=True, hide=True)
+        if install_path in toedit_fd.stdout:
+            new_rpath = os.path.relpath(target_rpath_dd_folder, os.path.dirname(file))
+            if macos:
+                patch_exit = 0
+                while not patch_exit:
+                    patch_exit = ctx.run(
+                        f"install_name_tool -rpath {install_path}/embedded/lib @loader_path/{new_rpath}/embedded/lib {file}",
```

Comment:
It may not be an issue in practice, but I'd rather not hardcode the `/embedded/lib` of the path here.